### PR TITLE
AB#1122897 Add enable custom auth config

### PR DESCRIPTION
**Pull request by ahmadfaridulazmi**
AB#1122897

With these changes, you can enable the custom auth on location service as below
```ruby
Volcanic::Location::Config do |config|
  config.enable_custom_auth = true
end
```

### CODE REVIEW

**Review by matthew-carter-access — APPROVED** [review]

File: lib/volcanic/location/config.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ def self.configure
     class Configuration
       class << self
         attr_accessor :authentication
+        attr_writer :enable_custom_auth
```
Comment (aljadepalaran) [inline lib/volcanic/location/config.rb:16]: can we merge the two `attr_writer` into one?

File: lib/volcanic/location/config.rb [hunk 2]
```diff
@@ -21,6 +22,10 @@ def domain_url
           @domain_url
         end
 
+        def enable_custom_auth
+          @enable_custom_auth == true
```
Comment (aljadepalaran) [inline lib/volcanic/location/config.rb:26]: can we clarify the purpose of this?

it looks like it's defined as a variable but acts more as a check, maybe a question mark might be better? `enable_custom_auth?`

Comment (ahmadfaridulazmi) [reply lib/volcanic/location/config.rb:25]: yes, is more of a placeholder to check if that setting is enabled

**Review by aljadepalaran — APPROVED** [review]
lgtm

**Pull request merged**